Translations of `Infra Editor` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Infra Editor](https://hosted.weblate.org/projects/osrd/infra-editor/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/infra-editor/horizontal-auto.svg)

flomonster committed:
front: infra-editor: translated using weblate (French)

Currently translated at 97.7% (385 of 394 strings)

Translation: OSRD/Infra Editor
Translate-URL: https://hosted.weblate.org/projects/osrd/infra-editor/fr/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>

mansil committed:
front: infra-editor: add translation using weblate (Portuguese)

Signed-off-by: Manuela Silva <mmsrs@sky.com>

mansil committed:
front: infra-editor: translated using weblate (Portuguese)

Currently translated at 6.3% (25 of 394 strings)

Translation: OSRD/Infra Editor
Translate-URL: https://hosted.weblate.org/projects/osrd/infra-editor/pt/

Signed-off-by: Manuela Silva <mmsrs@sky.com>

Khoyo committed:
front: infra-editor: translated using weblate (French)

Currently translated at 99.2% (391 of 394 strings)

Translation: OSRD/Infra Editor
Translate-URL: https://hosted.weblate.org/projects/osrd/infra-editor/fr/

Signed-off-by: Khoyo <khoyobegenn@gmail.com>

clarani reviewed (commented):
LGTM :white_check_mark:

clarani reviewed (approved):
Thanks for this PR :pray:

Uriel-Sautron commented on front/public/locales/fr/infraEditor.json:
```diff
@@ -360,18 +360,34 @@
         "description": "Début des bp/cc zone neutre"
+      },
+      "rev": {
+        "description": "REV de la section neutre bp/cc",
```
```suggestion
        "description": "Pancarte réversibilité",
```

Uriel-Sautron commented on front/public/locales/fr/infraEditor.json:
```diff
@@ -464,6 +490,9 @@
         "title": "Angle"
       },
+      "direction": {
+        "description": "Orientation du panneau sur la piste"
```
```suggestion
        "description": "Orientation du panneau sur la voie"
```

flomonster committed:
front: infra-editor: translated using weblate (French)

Currently translated at 99.2% (391 of 394 strings)

Translation: OSRD/Infra Editor
Translate-URL: https://hosted.weblate.org/projects/osrd/infra-editor/fr/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>
